entity-store/src/main/kotlin/jetbrains/exodus/entitystore/orientdb/OPersistentEntityStore.kt

```diff
@@ -65,7 +65,7 @@ class OPersistentEntityStore(
         val txn = session.begin().transaction
         session.activateOnCurrentThread()
 
-        currentTx = OStoreTransactionImpl(session, txn, this, schemaBuddy)
+        currentTx = OStoreTransactionImpl(session, txn, this, schemaBuddy, databaseProvider::acquireSession)
```

Comment:
Do we need `session` as param here?

Reply:
we need. sometimes it's session created before and managed by PersistentStore